srtcore/api.cpp

```diff
@@ -524,7 +524,7 @@ int CUDTUnited::newConnection(const SRTSOCKET listen, const sockaddr_any& peer,
    if (ls->m_pQueuedSockets->size() >= ls->m_uiBackLog)
    {
        w_error = SRT_REJ_BACKLOG;
-       LOGC(mglog.Error, log << "newConnection: listen backlog=" << ls->m_uiBackLog << " EXCEEDED");
+       LOGC(mglog.Note, log << "newConnection: listen backlog=" << ls->m_uiBackLog << " EXCEEDED");
```

Comment:
This is demoted up to Note because this doesn't happen during any connection processing or transmission. It is reported exclusively on the listener as a consequence of an excessive number of incoming connections, which is something on which this listener has no influence.

Reply:
Maybe `Warn` instead of `Note`?

Reply:
This one I'd still prefer as Note. This is because with the Warn level this will never appear when the connection didn't even start processing, while this event shouldn't be interested for the listener as it didn't actually take any action in response to the attempted connection.

Reply:
Not sure about it, so let it be "Note" then, like you suggest.